README.md

```diff
@@ -43,61 +43,47 @@ While our system does generate discoveries, these discoveries are not being disp
+### The Assignment
 
 Here are the asks we have for you:
 
-**Base Challenge: Acceptance Criteria**
+### 1. Base Challenge: Acceptance Criteria
 
 1. As a user, I want to see the full list of 100 discoveries, displayed in reverse-chronological order (most recent first).
-   - Please use the included mocks as a guideline - however, we don't expect exact, pixel-perfect replication of the mock!
-   - We suggest using the following icons and colors:
-     1. Discoveries of `type` `physiology` can use the icon `<MdFavoriteBorder />` set to color `primaryMain`
-     2. Discoveries of `type` `brain` can use the icon `<LuBrain />` set to color `errorLight`
-     3. Discoveries of `type` `medication` can use the icon `<LuPill />` set to color `successMain`
-2. As a user, I want the ability to accept:
-   - A single discovery
-   - Multiple discoveries
-   - All discoveries  
-     so that I can indicate which discoveries have been reviewed for the patient.
-3. As a user, I should be able to select one, multiple, or all discoveries.
-4. When a discovery is moved to the “Acknowledged” list:
-   - A “Submit” button should appear.
-   - Clicking “Submit” should save the list of acknowledged discoveries.
-   - The acknowledged discoveries list should persist after a page refresh.
-
-Once you have completed all the requirements above, open a PR with your submission.
-
-**Enhancement: Acceptance Criteria.**
-Please select one Enhancement from the list of Enhancements below to complete. Once you have selected an Enhancement, create a new branch. This new branch will contain your solution for your selected enhancement. In your PR, please state which Enhancement you selected.
-
-**Notes:**
-Once you create a UI where discoveries can be viewed, the “Acknowledged” list, and one enhancement, the take home challenge is considered complete and can be submitted.
-Please make note of the following:
+   - Please use the mocks below as a guideline - however, we don't expect exact, pixel-perfect replication of the mock!
+   - You can use the following icons and colors based on discovery `type`:
+     - physiology: `<MdFavoriteBorder />` set to color `primaryMain`
+     - brain: `<LuBrain />` set to color `errorLight`
+     - medication: `<LuPill />` set to color `successMain`
+2. As a user, I should be able to select one or multiple discoveries so that I can indicate which discoveries have been reviewed for the patient.
+3. When at least one discovery is selected, the “Submit” button should be enabled.
+4. Clicking “Submit” should move all selected discoveries to the "Acknowledged" list.
+5. The acknowledged discoveries list should persist after a page refresh.
+
+#### Mocks:
 
-1. The discoveries data already exists, and an endpoint already exists that will send back the list of discoveries. Neither the data nor the endpoint need to be updated in order for you to successfully complete the challenge, but you may modify the data or the endpoint should you choose to.
```

Comment:
moved these points to an "Additional Notes" section below (basically grouped the list of enhancements with the "Enhancements" title)